GEN-CHECK

```diff
@@ -1 +1 @@
-GEN_CHECK=(18.12.0=lts)
+GEN_CHECK=(18.12.0 14.21.0)
```

Comment:
The lts tag was forgotten as well as the other versions this PR builds.

Reply:
Ah. I didn't include lts because I thought it was already specified from a prior release in the ALIAS file. Fixing